src/styles/style.css

```diff
@@ -15,6 +17,15 @@ header h1 {
   transform: translate(-50%, -50%);
 }
 
+#vr-port {
+  height: 100% !important;
+  width: 100% !important;
+  position: absolute;
+  left: 0;
+  top: 0;
+  z-index: -1;
+}
+
 header h1,
 header::after {
   color: white;
```

Comment:
@jonesjonathan, @evan-brass In header h1, header::after add **position: relative** and **bottom: -150px**. I think this would fix aspect ratio and positioned relative to its normal position.